test/extended/deployments/deployments.go

```diff
@@ -451,6 +570,23 @@ var _ = g.Describe("deploymentconfigs", func() {
 	})
 
 	g.Describe("rolled back", func() {
+		g.AfterEach(func() {
```

Comment:
This is pretty close to the point where it needs to be abstracted - you added a few hundred lines of duplication.